canvas_manage_course/requirements/base.txt

```diff
@@ -13,5 +13,5 @@ requests==2.9.1
 
 git+ssh://git@github.com/penzance/canvas_python_sdk.git@v0.8.4#egg=canvas-python-sdk==0.8.4
 git+ssh://git@github.com/Harvard-University-iCommons/django-auth-lti.git@v1.2.5#egg=django-auth-lti==1.2.5
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.12.1#egg=django-icommons-common[async]==1.12.1
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.0#egg=django-icommons-ui==1.0
\ No newline at end of file
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.13#egg=django-icommons-common[async]==1.13
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.2.1#egg=django-icommons-ui==1.2.1
```

Comment:
Note to make this work at the moment you'll have to pull Harvard-University-iCommons/django-icommons-common#157 and https://github.com/Harvard-University-iCommons/django-icommons-ui/pull/82 locally (or reference the branches from github), as the tags haven't been created yet.